### ISSUE TITLE
Not skale and drag Sunflower in window manager. Without DE.

### ISSUE THREAD

**Issue by sania405** [MeanEYE/Sunflower#429]
I do not speak poisonously, and do not reproach. I am outraged that you say that the problem with mounting is with me, not with Sunflower. And only now, after starting on two systems, you acknowledge the problem.
With minimal installation, Debian with Horg and the window manager, the sunflower opens in a large size, and does not respond to dragging it to the side. I installed Openbox as well. The sunflower window does not move there either. All other functions work.

**Comment by MeanEYE** [comment]
Can you provide screen shot of this behavior?

**Comment by sania405** [comment]
This is Icewm. Sunflower is deployed to the whole ekra, does not move, the window of the Sunflower does not respond to the right tap. In this case, all the functions of the file manager work. In Openbox, the problem is similar.
![Снимок экрана от 2020-06-21 17-45-28](https://user-images.githubusercontent.com/67229047/85227659-89446680-b3e7-11ea-9191-4bb71c8ad39f.png)

**Comment by sania405** [comment]
![Снимок экрана от 2020-06-21 17-51-42](https://user-images.githubusercontent.com/67229047/85227738-e93b0d00-b3e7-11ea-922d-7a77d7b26bf6.png)

**Comment by MeanEYE** [comment]
Is there any output to console when you start it using `sunflower` command?

**Comment by sania405** [comment]
Not. In prewiew, you can see, have window terminal Sarura. Hi moved. Sunflower not мoved.In terminal not view bug, Sunflower turns on immediately after the command.
![Снимок экрана от 2020-06-21 17-56-26](https://user-images.githubusercontent.com/67229047/85227950-205dee00-b3e9-11ea-9da0-5b70ae9b317f.png)

**Comment by sania405** [comment]
look its      https://radikal.ru/video/pOJXngpGDju

**Comment by ArseniyK** [comment]
Can you provide what version of Sunflower, WM and OS you use

**Comment by sania405** [comment]
This is Debian 10, Sunflower 0.3.
Sunflower 0.4 does not start at all. I spoke about this earlier.

**Comment by ArseniyK** [comment]
Its look like known issue which many wm has with  pygtk and java progman
can you promt in terminal "wmname LG3D"  than run sunflower

**Comment by sania405** [comment]
Not. Does work.

**Comment by MeanEYE** [comment]
@ArseniyK did you manage to reproduce this issue? It might be that Sunflowers requested window size is bigger than screen size and window manager simply doesn't know what to do with this.

**Comment by sania405** [comment]
> @ArseniyK did you manage to reproduce this issue? 

Apparently, no ... I installed the utility offered to me, and also installed two file managers, XFE and 4panе
![Снимок экрана от 2020-06-24 21-47-19](https://user-images.githubusercontent.com/67229047/85616455-6b386980-b666-11ea-853c-b370b9277e52.png)
. Their windows are obedient, and scale perfectly. And move.

**Comment by MeanEYE** [comment]
Am not sure how to fix this issue. Sunflower doesn't have minimum geometry. However it does set 960x550 size by default when new configuration file is created. @sania405 could you open `~/.config/sunflower/windows.json` and set `main` geometry to something smaller and see if it works for you?

**Comment by sania405** [comment]
No. Install Virtualbox for yourself, and a mini deb-system with a window manager, and try it yourself. Wish you luck.
